docs/proposals/export-submission-review-fields.md

```diff
@@ -0,0 +1,20 @@
+# Export with submission review fields
```

Comment:
Could we also add the documentation in https://github.com/onaio/onadata/blob/master/docs/forms.rst? See https://api.ona.io/static/docs/forms.html#include-labels-in-csv-savzip-xls-and-zipped-csv-exports or https://github.com/onaio/onadata/blob/master/docs/forms.rst#include-labels-in-csv-savzip-xls-and-zipped-csv-exports of how we have done this before.

Reply:
done